xbmc/windowing/gbm/drm/DRMCrtc.h

```diff
@@ -30,6 +30,7 @@ class CDRMCrtc : public CDRMObject
   uint32_t GetX() const { return m_crtc->x; }
   uint32_t GetY() const { return m_crtc->y; }
   drmModeModeInfoPtr GetMode() const { return &m_crtc->mode; }
+  int GetModeValid() const { return m_crtc->mode_valid; }
```

Comment:
```suggestion
  bool GetModeValid() const { return static_cast<bool>(m_crtc->mode_valid); }
```

Reply:
done

Reply:
Instead of casting to bool, use comparison (e.g. `m_crtc->mode_valid != 0`). I could be wrong, but I think `static_cast<bool>(1) != static_cast<bool>(2)`

Reply:
Yes `m_crtc->mode_valid != 0` is the right solution here 👍 sorry for misleading.

Reply:
thanks, changed now to comparison